src/callbacks_stage/positivity_zhang_shu.jl

```diff
@@ -18,24 +18,26 @@ using the associated `thresholds` to determine the minimal acceptable values.
 The order of the `variables` is important and might have a strong influence
 on the robustness.
 """
-struct PositivityPreservingLimiterZhangShu{N, Thresholds<:NTuple{N,<:Real}, Variables<:NTuple{N,Any}}
-  thresholds::Thresholds
-  variables::Variables
+struct PositivityPreservingLimiterZhangShu{N, Thresholds <: NTuple{N, <:Real},
+                                           Variables <: NTuple{N, Any}}
+    thresholds::Thresholds
+    variables::Variables
 end
 
 function PositivityPreservingLimiterZhangShu(; thresholds, variables)
-  PositivityPreservingLimiterZhangShu(thresholds, variables)
+    PositivityPreservingLimiterZhangShu(thresholds, variables)
 end
 
-
-function (limiter!::PositivityPreservingLimiterZhangShu)(
-    u_ode, integrator, semi::AbstractSemidiscretization, t)
-  u = wrap_array(u_ode, semi)
-  @trixi_timeit timer() "positivity-preserving limiter" limiter_zhang_shu!(
-    u, limiter!.thresholds, limiter!.variables, mesh_equations_solver_cache(semi)...)
+function (limiter!::PositivityPreservingLimiterZhangShu)(u_ode, integrator,
+                                                         semi::AbstractSemidiscretization,
+                                                         t)
+    u = wrap_array(u_ode, semi)
+    @trixi_timeit timer() "positivity-preserving limiter" limiter_zhang_shu!(u,
+                                                                             limiter!.thresholds,
+                                                                             limiter!.variables,
+                                                                             mesh_equations_solver_cache(semi)...)
```

Comment:
I guess these are more than 92 characters. What about
```suggestion
    @trixi_timeit timer() "positivity-preserving limiter" begin
        limiter_zhang_shu!(u, limiter!.thresholds, limiter!.variables,
                           mesh_equations_solver_cache(semi)...)
    end
```
? If we agree that this may be better, we should probably also look at other places - `rhs!` may be a good candidate, I guess